src/commands/admin/sudo.ts

```diff
@@ -1,16 +1,38 @@
-import { Message } from 'discord.js';
-import { botMasterPerms } from '@lib/permissions';
+import { ApplicationCommandOptionData, ApplicationCommandPermissionData, CommandInteraction, Message } from 'discord.js';
+import { BOTMASTER_PERMS } from '@lib/permissions';
 import { getCommand } from '@root/src/lib/utils';
 import { Command } from '@lib/types/Command';
 
 export default class extends Command {
 
-	description = 'Allows you always to run other commands.';
-	extendedHelp = 'Sudo bypasses permission checks, disabled command checks and command location checks.';
-	usage = '<command> [args]';
+	description = 'Allows you always to run other commands by overriding permission, ' +
+	'disabled command, and command location checks.';
+	enabled = false;
+	tempPermissions: ApplicationCommandPermissionData[] = [BOTMASTER_PERMS];
 
-	async permissions(msg: Message): Promise<boolean> {
-		return await botMasterPerms(msg);
+	options: ApplicationCommandOptionData[] = [{
+		name: 'command',
+		description: 'The command you would like to sudo run.',
+		type: 'STRING',
+		required: true
+	},
+	{
+		name: 'args',
+		description: 'The args for the sudo-run command, ' +
```

Comment:
I... don't think this works with interactions?

Reply:
tis why we disabled the command good sir